homeassistant/components/nest/__init__.py

```diff
@@ -176,7 +180,7 @@ def handle_event(self, event_message: EventMessage):
         # This event triggered an update to a device that changed some
         # properties which the DeviceManager should already have received.
         # Send a signal to refresh state of all listening devices.
-        dispatcher_send(self._hass, SIGNAL_NEST_UPDATE)
+        async_dispatcher_send(self._hass, SIGNAL_NEST_UPDATE)
```

Comment:
Background on this change: the SDM integration runs inside the event loop, and so do it's callbacks. So the callbacks should use the async versions of functions.

Reply:
Tagged this PR for 117 because of this change.